Sources_v3/Workers/ChannelUpdater.swift

```diff
@@ -10,12 +10,18 @@ class ChannelUpdater<ExtraData: ExtraDataTypes>: Worker {
     ///
     /// - Parameters:
     ///   - channelQuery: The channel query used in the request
+    ///   - channelCreatedCallback: For some type of channels we need to obtain id from backend.
+    ///   This callback is needed to notify caller that channel is created or found and `id` is receivied.
```

Comment:
suggestion: `This callback is called with the obtained cid before the channel payload is saved to the DB.`